pandas/core/frame.py

```diff
@@ -9395,7 +9424,11 @@ def resample(
         )
 
     def to_timestamp(
-        self, freq=None, how: str = "start", axis: Axis = 0, copy: bool = True
+        self,
+        freq: Optional[str] = None,
```

Comment:
this is a frequency type, I think we have a type for it

Reply:
I don't think we do in typing, but I made one